torch/legacy/nn/MarginRankingCriterion.py

```diff
@@ -60,7 +60,7 @@ def updateGradInput(self, input, y):
             self.mask = self.mask.resize_as_(input[0]).copy_(dist)
             mask = self.mask
 
-            torch.ge(dist, 0, out=mask)
+            torch.ge_t(dist, 0, out=mask)
```

Comment:
The  gt_t versions produce Real outputs while the gt versions produce bools. Previously they were overloaded on return type, which was not going to fly in C++, and is generally a bad idea, so we moved them to a separate name.